### ISSUE TITLE
lzss has functions compress/decompress, not encode/decode and tests fail when this is corrected

### ISSUE THREAD

**Issue by juliangilbey** [chrislit/abydos#262]
In these lines of code: https://github.com/chrislit/abydos/blob/9ec382485394070cca186e9705c30583877dc8e1/abydos/distance/_ncd_lzss.py#L84-L87 the function `lzss.encode` is called.  However, when I compiled lzss myself, I discovered that the function is actually `lzss.compress`: see the source code at https://github.com/yyogo/pylzss/blob/871ef5a3dd188603801f4d93e5cce6ab51cafecf/pylzss.c#L262

Easy fix: change these four occurrences of `lzss.encode` to `lzss.compress`.

**Comment by juliangilbey** [comment]
(BTW, the `test.py` script included with PyLZSS has the same bug.  I've reported it there, too.)

**Comment by juliangilbey** [comment]
I've now patched this in my local code, but the two LZSS tests then fail:
```
======================================================================
FAIL: test_ncd_lzss_dist (tests.distance.test_distance_ncd_lzss.NCDlzssTestCases
)
Test abydos.distance.NCDlzss.dist.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/jdg/debian/spyder-packages/abydos/build-area/abydos-0.5.0+git20200210.9ec3824/.pybuild/cpython3_3.9/build/tests/distance/test_distance_ncd_lzss.py", line 51, in test_ncd_lzss_dist
    self.assertAlmostEqual(self.cmp.dist('Nigel', 'Niall'), 0.8333333333)
AssertionError: 1.0 != 0.8333333333 within 7 places (0.16666666669999997 difference)

======================================================================
FAIL: test_ncd_lzss_sim (tests.distance.test_distance_ncd_lzss.NCDlzssTestCases)
Test abydos.distance.NCDlzss.sim.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/jdg/debian/spyder-packages/abydos/build-area/abydos-0.5.0+git20200210.9ec3824/.pybuild/cpython3_3.9/build/tests/distance/test_distance_ncd_lzss.py", line 73, in test_ncd_lzss_sim
    self.assertAlmostEqual(self.cmp.sim('Nigel', 'Niall'), 0.1666666667)
AssertionError: 0.0 != 0.1666666667 within 7 places (0.1666666667 difference)
```

I tried running the code by hand, and this is what I found:
```python
$ python3
Python 3.9.1 (default, Dec  8 2020, 07:51:42) 
[GCC 10.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import lzss
>>> src = 'Nigel'
>>> tar = 'Niall'
>>> lzss.compress(src)
b'\x1fNigel'
>>> lzss.compress(tar)
b'\x1fNiall'
>>> lzss.compress(src+tar)
b'\xffNigelNia\x03ll'
>>> lzss.compress(tar+src)
b'\xffNiallNig\x03el'
>>> 
```
(so the lengths are 6, 6, 12, 12 respectively.)

It makes me wonder whether the LZSS library changed since you wrote this test code?

**Comment by chrislit** [comment]
It looks like PyLZSS was switched to completely different code & a new API back in July for version 0.3. This is all fixed in the 0.6.0c branch of Abydos.
